src/pages/admin/OrganizationRequest.tsx

```
@@ -0,0 +1,452 @@
+            >
+              {/* make the height of textarea 200px */}
+
+              <Input.TextArea style={{height : '200px'}} placeholder="Enter the reason for rejection" />
+            </Form.Item>
+          </Form>
+        </div>
+      ),
+      // Hide default OK button and use custom actions
+      footer: [
+        // align the buttons to the right
+        <Space style={{ float: 'right' }}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)